charts/images.yaml

```diff
@@ -278,11 +278,11 @@ images:
 - name: istio-proxy
   sourceRepository: github.com/istio/istio
   repository: gcr.io/istio-release/proxyv2
-  tag: "1.10.2-distroless"
+  tag: "1.12.0-distroless"
```

Comment:
Isn't there any change required to the istio CRDs (`./charts/istio/istio-crds/templates/crd-all.gen.yaml`)? I can see that previous upgrade PRs were actively modifying/synching this 5k lines long CRDs file. Can you explain how this CRDs file is maintained? Is it a copy of https://github.com/istio/istio/blob/1.12.0/manifests/charts/base/crds/crd-all.gen.yaml?

Reply:
It's a copy for the file at least it was for the last update. In the initial commit were also some crds removed which no longer exist.